### ISSUE TITLE
DoES PPE FAQs 

### ISSUE THREAD

**Issue by afroleft** [DoESLiverpool/covid19#55]
@johnmckerrell hit me with some questions that had been raised about why we were charging NHS providers for PPE Visors, so pulled together an FAQ to explain vs. explaining over and over to each person that asks. 

--- 

### Do you charge for PPE Visors? 
 
We receive orders from a number of different people: 
1. Panicked healthcare workers
2. Procurement  (buying teams)
3. Healthcare leadership teams

If procurement teams place orders, we ask them if they can pay to cover the costs of our production. Since ultimately, this helps us buy more materials and equipment to create more PPE visors, our North Star. 

For all other orders, we use the donation money to pay for the visors.

### What do you spend the money on?

* Materials we need to make visors
* Maintaining the machinery we use
* To package and deliver the items to hospitals and independent healthcare workers.
* To pay a modest living wage to the people working day and night on this [(link)](https://github.com/DoESLiverpool/covid19/issues/10#issuecomment-605910773). Many of our volunteers are freelancers and artists who have seen their business dry up over the last few weeks, this helps them to keep a roof over their heads.

### How do you intend to scale up?

* We have made a design which is must faster to produce than alternatives. [Link](https://www.businesscloud.co.uk/news/laser-cut-visors-for-nhs-staff-take-just-two-minutes-to-prod)
* We coordinate. Connecting our backlog of orders with partner producers (friends of DoES Liverpool) and Third Party Producers to ramp up supply of PPE Visors
* We are also purchasing new machinery to ramp up production. [Link](https://twitter.com/embedded_iot/status/1247558884056805376)

### What happens with the money that’s left over?

We will run a vote on the best charity to support with the money left over. Local food banks, Covid related charities, International Water Aid—all will be considered.

@zarino can we get these added to the website somewhere? Maybe on a separate view, that we can link to from GoFundMe. Ideas welcome

**Comment by Robotorium** [comment]
I'm a bit concerned about some of the wording on the website - particularly these two quotes: "We have a team of 20+ people working 24/7 producing 10,000 visors a week”. “Our laser-cut polypropylene visors are already being used by hospitals and GP surgeries across the North West, including Aintree Hospital, Manchester Royal Infirmary (MRi), Liverpool Royal, Walton Hospital, and East Lancashire Hospital."

From what I understand, and I'm currently stuck at home until next week so I might be misunderstanding, but I don't believe we're producing 10,000 visors a week. Also, have all of those places already received visors from us?

Obviously my apologies if I've got that wrong, but I'm worried because it would be incredibly bad PR if we were taking donations based on misinformation, however well-intentioned.

It's great to make DoES look good, but I think it's a lot more important to accurately portray what we're doing, especially when people are giving us money on that basis.

**Comment by afroleft** [comment]
> I'm a bit concerned about some of the wording on the website

Yep, we're aware of this. Zarino rolled out an update to the website that went out last night with next to no help, he's done a stellar job. A few wires crossed on where we hope to reach vs where we are today. He tagged me in the PR, so I should have picked this up, but I missed it.

After re-reading, we pushed an [update to correct this.](https://ppe.doesliverpool.com)

> It's great to make DoES look good, but I think it's a lot more important to accurately portray what we're doing, especially when people are giving us money on that basis.

Don't think anyone's disagreeing with this. 

What's happened here, is we're all moving very quickly to get stuff done, with people chipping in to support the whole operation (#47) and every now and then, we're going to get something wrong. 

Personally fine with that, so long as we course correct as quickly as possible. 

Laying the train track as the train is moving along is always going to be a bit messy around the edges.

**Comment by johnmckerrell** [comment]
This is great, thanks @afroleft. I have a few suggestions.

Not a fan of "our North Star" think something like "our reason for existing", "our first priority" or "our main aim" would be clearer.

> Most of which are freelancers, who have seen their businesses dry up over the last few weeks. This keeps beans on toast on the table.

How about "Many of our volunteers are freelancers and artists who have seen their business dry up over the last few weeks, this helps them to keep a roof over their heads." I wasn't quite sure how to end that but I'm not a big fan of the beans on toast bit, I think it's a bit too quirky for here.

> What happens with the money that’s left over?

Thought "any money" instead of "the money" might just make it sound less like we expect to have a big surplus.

**Comment by amcewen** [comment]
@afroleft I agree that it's a fast-moving situation, and there will be things that aren't right.  How about we start erring on the underpromise and overdeliver side?  Thanks for updating it, but I mentioned that copy to you late this morning and it's only been fixed after @Robotorium's comment this evening.

**Comment by afroleft** [comment]
> How about "Many of our volunteers are freelancers and artists who have seen their business dry up over the last few weeks, this helps them to keep a roof over their heads." 

@johnmckerrell — lovely edit thanks John. Agree, beans were a bit out there. 

> Thanks for updating it, but I mentioned that copy to you late this morning and it's only been fixed after @Robotorium's comment this evening.

@amcewen yes I made the copy amends locally forgot to push. When [this was raised](https://github.com/DoESLiverpool/covid19/issues/55#issuecomment-611125800), refreshed the browser and realised I hadn't deployed. Spinning lots of plates this end too. 

> How about we start erring on the underpromise and overdeliver side?

I can see that there's a culture clash emerging. I work in fast growth startups and yes we work with the press, raise VC money, scale teams and build products. DoES vibe is not about that 100%. I understand. 
 
Production is in your hands Mc, so you play this as you will. 

With this in mind, I suggest someone else takes over the marketing coordination so that I can step back from it. Hannah Forbes is working the social media, but she needs someone to coordinate with, edit the stuff she creates. 

Ideally someone from the DoES side can roll their sleeves up and take this on. That way you can make sure what goes out is a fit with the DoES culture.

**Comment by huffeec** [comment]
On the original question about charging..
The crowdfunder has done incredibly well, I don't know how much is already committed, but it looks like there's lots of materials budget.  A very rough calculation using 0.36/unit from the BOM every £1k is ~2700 laser cut units and that's about a week at DoES if the machines are staffed 24hrs 
That's a very conservative figure assuming everyone who makes visors needs their labour covering.
 
What proportion of the current orders are being paid for and which are being given free? I'm trying to guage how long we're expecting to be making these for. I know it's impossible to say when the infection rate will drop, or if/when more conventional supply chains get back to capacity, but it feels like we're in a position to be donating all of these?

**Comment by ajlennon** [comment]
>but it feels like we're in a position to be donating all of these?

@huffeec 

We have the option to supply a number of SMEs locally who have reached out to me this week. I've been holding off replying until I was sure there was no materials issue.

In addition the Design and Technology Association just issued guidance to schools nationally on how to engage with the PPE effort which means we can support every local school with a laser cutting capability.

So we're not limited by our own laser cutters and can deliver significantly more PPE as a local hub supporting capability in our area.

This is becoming the national model.

**Comment by johnmckerrell** [comment]
@afroleft I don't think you need to step down, I think what you're contributing is useful and quite frankly the project doesn't really have anyone else who can step in. It sounds like you are understanding the problem here. I wouldn't say that this was necessarily about the DoES culture, it just seems like for this project involving individuals contributing to the health of key workers that integrity is super important and we need to be very careful what we say.

**Comment by afroleft** [comment]
> What proportion of the current orders are being paid for and which are being given free?

`4` out of `20` are procurement orders. 

Conversation with @amcewen says that we want: 
* 6/7 people to run laser cutters (this might include other people that we know operating their machines)
* There are a few people working other roles that nearly equate to a 4/5 days per week, so rounding that off to `10` people 
* We have materials for around `40` days worth of production 

Back of the card but this is where you're landing without materials: 
<img width="577" alt="Screenshot 2020-04-09 at 10 08 02" src="https://user-images.githubusercontent.com/1761311/78878275-06e41180-7a4a-11ea-85fe-0d91ed21a71b.png">

> but it feels like we're in a position to be donating all of these?

Can you set out your working assumptions here @huffeec? 

I raised this [again and again](https://github.com/DoESLiverpool/covid19/issues/10#issuecomment-606120584) welcoming ideas but no one stepped up to make a call.

> I don't think you need to step down, I think what you're contributing is useful

Thanks @johnmckerrell but there are two key problems here: 

> health of key workers that integrity is super important and we need to be very careful what we say. 

Right there with you, but I'm mostly optimising for speed (clash). I'm happy to take on a chunk of risk, knowing that someone will put a foot wrong here and there (clash) and we can course correct as we go. If you look at the publish time of the GoFundMe, I raised it as an issue #36 and went ahead and published it 10 minutes after. Full of grammar and typos. By the time Zarino cleaned that up 2 hours later we'd already raised £1,400—that's how I prefer to work—ship, ship, ship, R&D in public. Appreciate there's a DoES ethos that works in different ways—to me they will be invisible strings.

To my mind, there's no clearer evidence that a culture clash has emerged when the actions of one, make people [genuinely worried](https://github.com/DoESLiverpool/covid19/issues/55#issuecomment-611125800) or one prompts Adrian to make a [snide remark](https://github.com/DoESLiverpool/covid19/issues/55#issuecomment-611157827) on a public forum when he could've picked up the phone to get to the bottom of it. I'm not here to make people worried or angry. That was never my intention. 

Still happy to support in other ways, just makes sense someone else works as the marketing editor from this point onwards to rubber stamp the fit.

**Comment by ajlennon** [comment]
For me, for what it's worth we have been doing this non-stop now for over two weeks.

We're tired. This is really challenging. We're starting to get grumpy and frazzled. And of course that manifests in being shorter with our peers than we otherwise would be.

We also come from different backgrounds and have different ideas about how projects should be approached and delivered.

What I want to say is that we are doing a brilliant job!

- We have Adrian's fantastic design -  doctors love it - we have orders
- We have Tom's marketing nouse and all the team built around that with Maz as FD running the financials and Hannah and Jack.
- We have people ringing around sorting producers and suppliers.
- We have materials now and we can make them for our orders.
- We don't really have any significant problems I am aware of today.

This is all frankly astonishing in such short space of time. 

Let's not lose sight of what we've achieved and are achieving through project fatigue. We are going to really make a difference to the NHS and our communities if we can just crack on?

Cheers.

**Comment by ajlennon** [comment]
>~2700 laser cut units and that's about a week at DoES if the machines are staffed 24hrs

@huffeec 

fwiw from my (very rough and may be wrong) calcs: I'm getting one sheet cut every 20 mins which gives 18 visor bands. On that basis our theoretical maximum with 24 hour shift is 1200 / day or 9000 / week (More if we can bring up the other laser cutters)

**Comment by drakard** [comment]
For what it's worth the BBC have stated products without the kitemark cannot be sold (for money).

> While templates are being shared and modified among the 3D-printer community for creating PPE kit, not one of the resulting products has a CE Kitemark, the European safety standard, which ultimately means that there is no quality benchmark.
>
>For this reason they cannot be sold - so no maker should be offering them in exchange for cash.
>
>https://www.bbc.co.uk/news/health-52201696

I know there is more sophisticated discussion already at https://github.com/DoESLiverpool/covid19/issues/24 and it's a voluntary contribution. I just thought I'd bring it onto the radar.

**Comment by afroleft** [comment]
> For what it's worth the BBC have stated products without the kitemark cannot be sold (for money).

@drakard Yes had been thinking about this from a liability angle too. 

To be 100% clear, I have no experience in commercial law, but my sense would be that our liabilities could be reduced by donating rather than selling.

**Comment by JackiePease** [comment]
On 11 Apr 2020 11:00, Tom Darlow <notifications@github.com> wrote:

For what it's worth the BBC have stated products without the kitemark cannot be sold (for money).

@drakard Yes had been thinking about this from a liability angle too.
To be 100% clear, I have no experience in commercial law, but my sense would be that our liabilities could be reduced by donating rather than selling.

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.

We could do with a clear idea of how long we could sustain that for, given we have reached our stretch fundraising target.
